shopgate/pwa
# Replaced deprecated FR, LV, FE and GE iso province codes

# Description

Replaced deprecated FR, LV, FE and GE iso province codes, used by the form builder with the current province codes
## Type of change

Please add an "x" into the option that is relevant:

- [  ] New Feature :rocket: (non-breaking change which adds functionality)
- [  ] Breaking Change :boom: (fix or feature that would cause existing functionality to not work as expected)
- [  ] Polish :nail_care: (Just some cleanups)
- [  ] Docs :memo: (Changes in the documentations)
- [  ] Internal :house: Only relates to internal processes.

## How to test it

Open up the address management page, using a test shop with the correct provinces and countries set up in one of the supporting extensions and check if everything shows up correctly.
